README.md

```diff
@@ -12,14 +12,21 @@ We welcome contributions from community members to this repo:
 - **New documentation**: If you contributed code in [dbt-core](https://github.com/fishtown-analytics/dbt), we encourage you to also write the docs here!
 - **Refactors**: At this time, we are unable to support community members who wish to re-write sections of docs.getdbt.com. We hope to change this in the future!
 
-If you are contributing significant changes, it may be worth setting up the repo to run locally, as follows:
-1. Ensure node is installed: `brew install node`
-2. Clone this repo: `git clone git@github.com:fishtown-analytics/docs.getdbt.com.git`
-3. `cd` into the repo: `cd docs.getdbt.com`
-3. And then `cd` into the `website` subdirectory: `cd website`
-4. Install the required node packages: `npm install`
-5. Build the website: `npm start`
-6. Before pushing your changes to a branch, check that all links work by using the `make build` script.
+### Running the Docs site locally
+
+We recommend locally rendering changes made to the docs site so you can review your proposed modifications. Our setup instructions use [homebrew](https://brew.sh/):
+
+0. If applicable, install [Xcode CLTs](https://developer.apple.com/download/more/); you'll likely need an AppleID for this.
+1. Install `node`: `brew install node`
+2. Install `yarn`: `brew install yarn`
```

Comment:
Are both needed? I was under the impression it was one or the other

Reply:
oops! It's only necessary if you attempt to solve the docusaurus issue, which I did. >.>

Removed--updated version being pushed. Good catch!